cpp/oneapi/dal/data/detail/undirected_adjacency_array_graph_impl.hpp

```diff
@@ -23,11 +23,21 @@
 
 namespace oneapi::dal::preview::detail {
```

Comment:
what is the style?
namespace oneapi:...::detail or
namespace oneapi {
namespace dal {
}
}

Reply:
any

Reply:
how is the rule formulated?